src/full_client.toit

```diff
@@ -217,6 +217,32 @@ interface ReconnectionStrategy:
   /** Closes the strategy, indicating that no further reconnection attempts should be done. */
   close -> none
 
+/**
+A latch that has a timeout when getting its value.
+*/
+monitor WaitLatch_:
```

Comment:
Does this need to be a latch? You don't need the value for anything and has_value doesn't help here. Maybe just turn this into a signal with a single boolean inside?

Reply:
done.